Jasmine/Tetris/ViewModels/TetrisGameViewModel.swift

```diff
@@ -90,9 +99,7 @@ class TetrisGameViewModel {
     }
 
     private func checkForMatchingPhrase(at coordinate: Coordinate, rowWise: Bool) -> Set<Coordinate>? {
-        guard let phraseLen = gameData.phrases.first?.chinese.count else {
-            fatalError("No phrases found")
-        }
+        let phraseLen = phraseLength
```

Comment:
Just slightly more efficient since .count is O(k) and phraseLen is called in a loop